packages/forms/test/form_group_spec.ts

```diff
@@ -981,6 +1013,419 @@ describe('FormGroup', () => {
          expect(g.errors).toEqual({'async': true});
          expect(g.get('one')!.errors).toEqual({'async': true});
        }));
+
+    it('should handle failing FormGroup validator resolving after successful child validator',
+       fakeAsync(() => {
+         const c = new FormControl(
+             'fcValue', null!, simpleAsyncValidator({timeout: 1, shouldFail: false}));
+         const g =
+             new FormGroup({'one': c}, null!, simpleAsyncValidator({timeout: 2, shouldFail: true}));
+
+         // Initially, the from group and nested control are in pending state
```

Comment:
There are few places with the same typo, could you please have a look?
```suggestion
         // Initially, the form group and nested control are in pending state
```

Reply:
I have updated all the occurrences of this typo. Thank you